lib/ansible/module_utils/network/cnos/cnos.py

```diff
@@ -2771,7 +2800,7 @@ def doSecureStartupConfigBackUp(
         username + "@" + server + "/" + confPath + " vrf management\n"
     # debugOutput(command)
     response = waitForDeviceResponse(command, "(yes/no)", 3, obj)
-    if(response.lower().find("error-101")):
+    if(response.lower().find("Error-101")):
```

Comment:
Not sure about this, I don't think it would ever match.

Could you please retest the error conditions

```python
>>> response="Bad Error-101"
>>> response.lower()
'bad error-101'
>>> response.lower().find("error-101")
4 # Found 
>>> response.lower().find("Error-101")
-1 # not found (as case doesn't match
>>> response.lower().find("foo-101")
-1 # not found

```

Reply:
Its a Bug. I will fix it, Thanks